dist/index.js

```diff
@@ -69901,13 +69901,28 @@ async function ScanPullRequest(
     for (let i = 0; i < sarif.runs[0].results.length; i++) {
       let results = sarif.runs[0].results[i];
       const commit_sha = results.partialFingerprints.commitSha;
+      const fingerprint =
+        commit_sha +
+        ":" +
+        results.locations[0].physicalLocation.artifactLocation.uri +
+        ":" +
+        results.ruleId +
+        ":" +
+        results.locations[0].physicalLocation.region.startLine;
 
       let proposedComment = {
         owner: owner,
         repo: repo,
         pull_number: eventJSON.number,
-        body: `🛑 **gitleaks** has detected a secret with rule-id \`${results.ruleId}\` in commit ${commit_sha}
-If this secret is a true positive, please rotate the secret ASAP.`,
+        body: `🛑 **Gitleaks** has detected a secret with rule-id \`${results.ruleId}\` in commit ${commit_sha}.
+If this secret is a _true_ positive, please rotate the secret ASAP.
+
+If this secret is a _false_ positive, please add the fingerprint below to your \`.gitleaksignore\` file and commit the change to this branch:
```

Comment:
consider replacing "please" with "you can"

Also I'd suggest replacing "this branch:" with "this branch." i.e. replace the colon with a period. The colon almost makes it seem like what follows is a branch.